app/Http/Controllers/FaultsController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\FaultsTable;
+use App\Role;
+use DB;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class FaultsController extends Controller
+{
+    public function index()
+    {
+        return view('faults.app');
+    }
+
+    public function addRecord(Request $new)
+    {
+        DB::table('faults')->insert(
+            [
+                'user_id' => Auth::User()->id,
+                'location' => $new['location'],
+                'description' => $new['description'],
+                'status' => FaultsTable::UNSEEN,
+                'created_at' => date('Y-m-d H:i:s'),
+                'updated_at' => date('Y-m-d H:i:s')
```

Comment:
Is the date format hardcoded everywhere?
Probably the majority of of the other countries would use a different date format.

Reply:
This date format isn't the same as the one being displayed. It shouldn't be visible to or affect any user wherever they are.

Reply:
Just a note, I think this is exactly the universal date format.
Also the same that MySQL accepts by default.

But, you should avoid using the DB facade whenever possible and use Laravel models. That way you do not have to worry about the timestamps, they will be updated automatically.